### PR TITLE
Clarify cron handling

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This tidies up how the configuration for the cron queue is handled so that the side-effects are more explicit. It also avoids loading the config twice.

### CODE REVIEW

File: django_lightweight_queue/runner.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from . import app_settings
 from .utils import set_process_title, get_backend
 from .worker import Worker
-from .cron_scheduler import CronScheduler, get_config
+from .cron_scheduler import CronScheduler, CRON_QUEUE_NAME, get_cron_config, ensure_queue_workers_for_config
```
Comment (tomokas) [inline django_lightweight_queue/runner.py]: Line length

File: django_lightweight_queue/cron_scheduler.py [hunk 2]
```diff
@@ -134,11 +137,15 @@ def get_matcher(minval, maxval, t):
...
 
-            # We must ensure we have at least one worker for this queue.
-            app_settings.WORKERS.setdefault(row['queue'], 1)
-
     return config
 
+
+def ensure_queue_workers_for_config(config):
+    for row in config:
+        # We must ensure we have at least one worker for this queue.
```
Comment (tomokas) [inline django_lightweight_queue/cron_scheduler.py]: The comment is a bit out-of-context now. How about `# We must ensure we have at least one worker for every queue specified`

Comment (PeterJCLaw) [inline django_lightweight_queue/cron_scheduler.py]: a5743f9; I realised that the comment is actually also wrong, hopefully the new docstring is clearer.

File: django_lightweight_queue/runner.py (hunk 1 above)
Comment (PeterJCLaw) [inline django_lightweight_queue/runner.py]: 12766d459a6b2d4314c3ba0514292d474e9429d8

**Comment by PeterJCLaw** [comment]
@tomokas I think this is now all fixed, please could you take another look (cc @danpalmer)

**Pull request merged**